### PR TITLE
Update italian translation part 2

### PR DESCRIPTION
**Pull request by serhack**
Enjoy them!

### CODE REVIEW

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
-In addition to all Boost and OpenSSL contributors, and all coders since time-immemorial who have made this project possible, we would like to acknowledge the following:
-
-- Wei Dai, Jeffrey Walton, and all the developers at Crypto++ for proving an influential, free C++ library of cryptographical library
-- Dean Michael Berris, Glyn Matthews, and all the developers at cpp-netlib for providing an actively developed cross-platform networking library
-- All developers involved in the [ref10 implementation from SUPERCOP](http://bench.cr.yp.to/supercop.html) from which we base our Ed25519 implementation
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: jhkjh

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
-- zzz, str4d, and all Java I2P developers both past and present who have set the standard for all other I2P implementations to follow
-- orion for providing ```i2pcpp```: the [original](http://git.repo.i2p.xyz/w/i2pcpp.git) C++ implementation of I2P
-- orignal for providing ```i2pd```: the most feature-complete C++ implementation of I2P for [us to fork from](https://github.com/purplei2p/i2pd/commit/45d27f8ddc43e220a9eea42de41cb67d5627a7d3)
-- EinMByte for improving all C++ I2P implementations (including Kovri)
+Oltre al ringraziamento a tutti i contribuitori dei progetti Boost e OpenSSL e tutti gli sviluppatori, vorremo ringraziare i seguenti:
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: repeted "ringraziamento"

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 3]
```diff
@@ -1,9 +1,8 @@
-In addition to all Boost and OpenSSL contributors, and all coders since time-immemorial who have made this project possible, we would like to acknowledge the following:
-
-- Wei Dai, Jeffrey Walton, and all the developers at Crypto++ for proving an influential, free C++ library of cryptographical library
-- Dean Michael Berris, Glyn Matthews, and all the developers at cpp-netlib for providing an actively developed cross-platform networking library
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: "actively developed cross-platform networking library" this part should be translated

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 4]
```diff
@@ -1,9 +1,8 @@
...
-- All developers involved in the [ref10 implementation from SUPERCOP](http://bench.cr.yp.to/supercop.html) from which we base our Ed25519 implementation
-- zzz, str4d, and all Java I2P developers both past and present who have set the standard for all other I2P implementations to follow
-- orion for providing ```i2pcpp```: the [original](http://git.repo.i2p.xyz/w/i2pcpp.git) C++ implementation of I2P
-- orignal for providing ```i2pd```: the most feature-complete C++ implementation of I2P for [us to fork from](https://github.com/purplei2p/i2pd/commit/45d27f8ddc43e220a9eea42de41cb67d5627a7d3)
-- EinMByte for improving all C++ I2P implementations (including Kovri)
+Oltre al ringraziamento a tutti i contribuitori dei progetti Boost e OpenSSL e tutti gli sviluppatori, vorremo ringraziare i seguenti:
+- Wei Dai, Jeffrey Walton, e tutti gli sviluppatori di Crypto++, dimostrando di aver implementato una libreria C++ sicura per la crittografia
+- Dean Michael Berris, Glyn Matthews, e tutti gli sviluppatori di cpp-netlib  per aver sviluppato una libreria network cross-platform networking library
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: "libreria network cross-platform networking library" - this part should be translated

Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: "libreria network cross-platform networking library" - this part should be translated

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 5]
```diff
@@ -1,9 +1,8 @@
...
+- Tutti gli sviluppatori coinvolti in the [ref10 implementation from SUPERCOP](http://bench.cr.yp.to/supercop.html) da cui basiamo la nostra implementazione Ed25519 
+- zzz, str4d, e tutti gli sviluppatori Java I2P (passati e presente) che hanno impostato una solida base per le implementazione che sono venute successivamente 
+- ororin per lo sviluppo di  ```i2pcpp```: l'implementaione [originale](http://git.repo.i2p.xyz/w/i2pcpp.git) di I2P
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: ororin should be orion

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 6]
```diff
@@ -1,9 +1,8 @@
...
-- orion for providing ```i2pcpp```: the [original](http://git.repo.i2p.xyz/w/i2pcpp.git) C++ implementation of I2P
-- orignal for providing ```i2pd```: the most feature-complete C++ implementation of I2P for [us to fork from](https://github.com/purplei2p/i2pd/commit/45d27f8ddc43e220a9eea42de41cb67d5627a7d3)
-- EinMByte for improving all C++ I2P implementations (including Kovri)
+Oltre al ringraziamento a tutti i contribuitori dei progetti Boost e OpenSSL e tutti gli sviluppatori, vorremo ringraziare i seguenti:
+- Wei Dai, Jeffrey Walton, e tutti gli sviluppatori di Crypto++, dimostrando di aver implementato una libreria C++ sicura per la crittografia
+- Dean Michael Berris, Glyn Matthews, e tutti gli sviluppatori di cpp-netlib  per aver sviluppato una libreria network cross-platform networking library
+- Tutti gli sviluppatori coinvolti in the [ref10 implementation from SUPERCOP](http://bench.cr.yp.to/supercop.html) da cui basiamo la nostra implementazione Ed25519 
+- zzz, str4d, e tutti gli sviluppatori Java I2P (passati e presente) che hanno impostato una solida base per le implementazione che sono venute successivamente
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: "presente" should be "presenti" and "per le implementazioni successive" should be added

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 7]
```diff
@@ -1,9 +1,8 @@
...
-- zzz, str4d, and all Java I2P developers both past and present who have set the standard for all other I2P implementations to follow
-- orion for providing ```i2pcpp```: the [original](http://git.repo.i2p.xyz/w/i2pcpp.git) C++ implementation of I2P
-- orignal for providing ```i2pd```: the most feature-complete C++ implementation of I2P for [us to fork from](https://github.com/purplei2p/i2pd/commit/45d27f8ddc43e220a9eea42de41cb67d5627a7d3)
-- EinMByte for improving all C++ I2P implementations (including Kovri)
+Oltre al ringraziamento a tutti i contribuitori dei progetti Boost e OpenSSL e tutti gli sviluppatori, vorremo ringraziare i seguenti:
+- Wei Dai, Jeffrey Walton, e tutti gli sviluppatori di Crypto++, dimostrando di aver implementato una libreria C++ sicura per la crittografia
+- Dean Michael Berris, Glyn Matthews, e tutti gli sviluppatori di cpp-netlib  per aver sviluppato una libreria network cross-platform networking library
+- Tutti gli sviluppatori coinvolti in the [ref10 implementation from SUPERCOP](http://bench.cr.yp.to/supercop.html) da cui basiamo la nostra implementazione Ed25519
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: should be clearer

File: kovri.i2p/_i18n/it/acknowledgements.md [hunk 8]
```diff
@@ -1,9 +1,8 @@
...
-- EinMByte for improving all C++ I2P implementations (including Kovri)
+Oltre al ringraziamento a tutti i contribuitori dei progetti Boost e OpenSSL e tutti gli sviluppatori, vorremo ringraziare i seguenti:
+- Wei Dai, Jeffrey Walton, e tutti gli sviluppatori di Crypto++, dimostrando di aver implementato una libreria C++ sicura per la crittografia
+- Dean Michael Berris, Glyn Matthews, e tutti gli sviluppatori di cpp-netlib  per aver sviluppato una libreria network cross-platform networking library
+- Tutti gli sviluppatori coinvolti in the [ref10 implementation from SUPERCOP](http://bench.cr.yp.to/supercop.html) da cui basiamo la nostra implementazione Ed25519 
+- zzz, str4d, e tutti gli sviluppatori Java I2P (passati e presente) che hanno impostato una solida base per le implementazione che sono venute successivamente 
+- ororin per lo sviluppo di  ```i2pcpp```: l'implementaione [originale](http://git.repo.i2p.xyz/w/i2pcpp.git) di I2P
+- original per lo sviluppo di ```i2pd```: l'implementazione più matura di I2P per [noi](https://github.com/purplei2p/i2pd/commit/45d27f8ddc43e220a9eea42de41cb67d5627a7d3)
```
Comment (erciccione) [inline kovri.i2p/_i18n/it/acknowledgements.md]: "original" should be "orignal"

**Comment by serhack** [comment]
@erciccione thanks :)

**Comment by anonimal** [comment]
Grazie! @serhack @erciccione 👍

**Pull request merged**